lighthouse-core/test/lib/url-shim-test.js

```diff
@@ -29,6 +29,12 @@ describe('URL Shim', () => {
     assert.doesNotThrow(_ => new URL(url));
   });
 
+  it.skip('handles URLs with multiple dashes', () => {
+    // from https://github.com/GoogleChrome/lighthouse/issues/1972
+    const url = 'https://r15---sn-o097znl7.googlevideo.com/generate_204?conn2';
+    assert.doesNotThrow(_ => new URL(url));
```

Comment:
I noticed you've been on a kick lately to banish `doesNotThrow` from our tests. Do you not think it enhances the clarity that the statement is being executed exclusively to check for side effects? There's a reason the function exists :)